### ISSUE TITLE
HLS support

### ISSUE THREAD

**Issue by D3vl0per** [LevitatingBusinessMan/twitch-drop-farm#1]
I just want to help you.

> Since Puppeteer (in all configurations) controls a desktop version of Chromium/Chrome, features that are only supported by the mobile version of Chrome are not supported. This means that Puppeteer does not support HTTP Live Streaming (HLS).
Source: https://github.com/puppeteer/puppeteer/blob/master/README.md

So:
![Screenshot_20200417_033946](https://user-images.githubusercontent.com/34040149/79522880-2b00ad80-805d-11ea-9982-2101038a7e9c.png)

**Comment by LevitatingBusinessMan** [comment]
Yes I saw this, but honestly I don't think twitch cares much if the video player is playing. I'll keep the issue open though.
